### PR TITLE
fix: preview deployment 

### PR DESCRIPTION
**Pull request by kevbarns**

### CODE REVIEW

**Comment by kevbarns** [comment]
🚀

**Review by moroine — APPROVED** [review]

**Comment by kevbarns** [comment]
🚀

File: .infra/docker-compose.preview-system.yml [hunk 1]
```diff
@@ -32,16 +32,15 @@ services:
     <<: *default
     image: nginxproxy/acme-companion
     container_name: nginx-proxy-acme
-    depends_on:
-      - reverse_proxy
     volumes:
+      - conf:/etc/nginx/conf.d
+      - vhost:/etc/nginx/vhost.d
+      - html:/usr/share/nginx/html
```
Comment (moroine) [inline .infra/docker-compose.preview-system.yml]: Pourquoi?

Comment (kevbarns) [reply .infra/docker-compose.preview-system.yml]: testing, j'ai revert

**Review by moroine — APPROVED** [review]

**Comment by kevbarns** [comment]
🚀

**Comment by kevbarns** [comment]
🚀

**Comment by kevbarns** [comment]
🚀

**Pull request merged**